 Thu, 05 Mar 2020 12:00:52  INFO logger.py <module> [ line:20 ] hehehehe
 Thu, 05 Mar 2020 12:00:52  ERROR logger.py <module> [ line:21 ] heheheheheheh
Traceback (most recent call last):
  File "D:\Program Files (x86)\python\lib\site-packages\_pytest\config\__init__.py", line 456, in _importconftest
    return self._conftestpath2mod[key]
KeyError: WindowsPath('D:/python/mystudy/AppFramework/TestCases/conftest.py')
 Thu, 05 Mar 2020 12:00:52  INFO conftest.py init_session [ line:65 ] xxxxxxx
 Thu, 05 Mar 2020 12:01:28  INFO test_0_login.py test_login_tele_wrong [ line:30 ] 手机号输入错误用例
 Thu, 05 Mar 2020 12:01:28  INFO bottom_widget.py switch_bottom_widget [ line:15 ] 切换到页面我
 Thu, 05 Mar 2020 12:01:31  INFO basepage.py visiable_ele [ line:38 ] 元素('-android uiautomator', 'new UiSelector().text("我")')查找成功，耗时2.3397891521453857s
 Thu, 05 Mar 2020 12:01:31  INFO basepage.py visiable_ele [ line:38 ] 元素('-android uiautomator', 'new UiSelector().text("我")')查找成功，耗时0.09662389755249023s
 Thu, 05 Mar 2020 12:01:31  INFO basepage.py get_ele [ line:56 ] 元素('-android uiautomator', 'new UiSelector().text("我")')查找成功！
 Thu, 05 Mar 2020 12:01:32  INFO basepage.py click_ele [ line:102 ] 元素('-android uiautomator', 'new UiSelector().text("我")')成功点击!
 Thu, 05 Mar 2020 12:01:33  INFO basepage.py visiable_ele [ line:38 ] 元素('id', 'com.xxzb.fenwoo:id/et_phone')查找成功，耗时1.1453511714935303s
 Thu, 05 Mar 2020 12:01:33  INFO basepage.py visiable_ele [ line:38 ] 元素('id', 'com.xxzb.fenwoo:id/et_phone')查找成功，耗时0.054656267166137695s
 Thu, 05 Mar 2020 12:01:33  INFO basepage.py get_ele [ line:56 ] 元素('id', 'com.xxzb.fenwoo:id/et_phone')查找成功！
 Thu, 05 Mar 2020 12:01:34  INFO basepage.py input_ele_text [ line:81 ] 元素('id', 'com.xxzb.fenwoo:id/et_phone')成功输入 
 Thu, 05 Mar 2020 12:01:35  INFO basepage.py visiable_ele [ line:38 ] 元素('id', 'com.xxzb.fenwoo:id/btn_next_step')查找成功，耗时0.835456132888794s
 Thu, 05 Mar 2020 12:01:35  INFO basepage.py get_ele [ line:56 ] 元素('id', 'com.xxzb.fenwoo:id/btn_next_step')查找成功！
 Thu, 05 Mar 2020 12:01:35  INFO basepage.py click_ele [ line:102 ] 元素('id', 'com.xxzb.fenwoo:id/btn_next_step')成功点击!
 Thu, 05 Mar 2020 12:01:37  INFO basepage.py visiable_ele [ line:38 ] 元素('id', 'com.xxzb.fenwoo:id/tv_message')查找成功，耗时0.09369111061096191s
 Thu, 05 Mar 2020 12:01:37  INFO basepage.py get_ele [ line:56 ] 元素('id', 'com.xxzb.fenwoo:id/tv_message')查找成功！
 Thu, 05 Mar 2020 12:01:37  INFO basepage.py get_ele_attr [ line:113 ] 获取元素('id', 'com.xxzb.fenwoo:id/tv_message')得text属性为无效的手机号
 Thu, 05 Mar 2020 12:01:37  INFO test_0_login.py test_login_tele_wrong [ line:37 ] Test Pass!
 Thu, 05 Mar 2020 12:01:37  INFO basepage.py visiable_ele [ line:38 ] 元素('id', 'com.xxzb.fenwoo:id/btn_confirm')查找成功，耗时0.054656267166137695s
 Thu, 05 Mar 2020 12:01:37  INFO basepage.py get_ele [ line:56 ] 元素('id', 'com.xxzb.fenwoo:id/btn_confirm')查找成功！
 Thu, 05 Mar 2020 12:01:37  INFO basepage.py click_ele [ line:102 ] 元素('id', 'com.xxzb.fenwoo:id/btn_confirm')成功点击!
 Thu, 05 Mar 2020 12:01:51  INFO test_0_login.py test_login_tele_wrong [ line:30 ] 手机号输入错误用例
 Thu, 05 Mar 2020 12:01:51  INFO bottom_widget.py switch_bottom_widget [ line:15 ] 切换到页面我
 Thu, 05 Mar 2020 12:01:54  INFO basepage.py visiable_ele [ line:38 ] 元素('-android uiautomator', 'new UiSelector().text("我")')查找成功，耗时2.78668475151062s
 Thu, 05 Mar 2020 12:01:54  INFO basepage.py visiable_ele [ line:38 ] 元素('-android uiautomator', 'new UiSelector().text("我")')查找成功，耗时0.5819816589355469s
 Thu, 05 Mar 2020 12:01:55  INFO basepage.py get_ele [ line:56 ] 元素('-android uiautomator', 'new UiSelector().text("我")')查找成功！
 Thu, 05 Mar 2020 12:01:55  INFO basepage.py click_ele [ line:102 ] 元素('-android uiautomator', 'new UiSelector().text("我")')成功点击!
 Thu, 05 Mar 2020 12:01:56  INFO basepage.py visiable_ele [ line:38 ] 元素('id', 'com.xxzb.fenwoo:id/et_phone')查找成功，耗时1.142895221710205s
 Thu, 05 Mar 2020 12:01:57  INFO basepage.py visiable_ele [ line:38 ] 元素('id', 'com.xxzb.fenwoo:id/et_phone')查找成功，耗时0.4133179187774658s
 Thu, 05 Mar 2020 12:01:57  INFO basepage.py get_ele [ line:56 ] 元素('id', 'com.xxzb.fenwoo:id/et_phone')查找成功！
 Thu, 05 Mar 2020 12:01:57  INFO basepage.py input_ele_text [ line:81 ] 元素('id', 'com.xxzb.fenwoo:id/et_phone')成功输入15264
 Thu, 05 Mar 2020 12:01:58  INFO basepage.py visiable_ele [ line:38 ] 元素('id', 'com.xxzb.fenwoo:id/btn_next_step')查找成功，耗时0.8605737686157227s
 Thu, 05 Mar 2020 12:01:58  INFO basepage.py get_ele [ line:56 ] 元素('id', 'com.xxzb.fenwoo:id/btn_next_step')查找成功！
 Thu, 05 Mar 2020 12:01:58  INFO basepage.py click_ele [ line:102 ] 元素('id', 'com.xxzb.fenwoo:id/btn_next_step')成功点击!
 Thu, 05 Mar 2020 12:02:00  INFO basepage.py visiable_ele [ line:38 ] 元素('id', 'com.xxzb.fenwoo:id/tv_message')查找成功，耗时0.0556337833404541s
 Thu, 05 Mar 2020 12:02:00  INFO basepage.py get_ele [ line:56 ] 元素('id', 'com.xxzb.fenwoo:id/tv_message')查找成功！
 Thu, 05 Mar 2020 12:02:00  INFO basepage.py get_ele_attr [ line:113 ] 获取元素('id', 'com.xxzb.fenwoo:id/tv_message')得text属性为无效的手机号
 Thu, 05 Mar 2020 12:02:00  INFO test_0_login.py test_login_tele_wrong [ line:37 ] Test Pass!
 Thu, 05 Mar 2020 12:02:00  INFO basepage.py visiable_ele [ line:38 ] 元素('id', 'com.xxzb.fenwoo:id/btn_confirm')查找成功，耗时0.04880213737487793s
 Thu, 05 Mar 2020 12:02:00  INFO basepage.py get_ele [ line:56 ] 元素('id', 'com.xxzb.fenwoo:id/btn_confirm')查找成功！
 Thu, 05 Mar 2020 12:02:01  INFO basepage.py click_ele [ line:102 ] 元素('id', 'com.xxzb.fenwoo:id/btn_confirm')成功点击!
 Thu, 05 Mar 2020 12:02:17  INFO test_0_login.py test_login_tele_wrong [ line:30 ] 手机号输入错误用例
 Thu, 05 Mar 2020 12:02:17  INFO bottom_widget.py switch_bottom_widget [ line:15 ] 切换到页面我
 Thu, 05 Mar 2020 12:02:19  INFO basepage.py visiable_ele [ line:38 ] 元素('-android uiautomator', 'new UiSelector().text("我")')查找成功，耗时1.6271038055419922s
 Thu, 05 Mar 2020 12:02:19  INFO basepage.py visiable_ele [ line:38 ] 元素('-android uiautomator', 'new UiSelector().text("我")')查找成功，耗时0.1054074764251709s
 Thu, 05 Mar 2020 12:02:19  INFO basepage.py get_ele [ line:56 ] 元素('-android uiautomator', 'new UiSelector().text("我")')查找成功！
 Thu, 05 Mar 2020 12:02:20  INFO basepage.py click_ele [ line:102 ] 元素('-android uiautomator', 'new UiSelector().text("我")')成功点击!
 Thu, 05 Mar 2020 12:02:21  INFO basepage.py visiable_ele [ line:38 ] 元素('id', 'com.xxzb.fenwoo:id/et_phone')查找成功，耗时1.0907680988311768s
 Thu, 05 Mar 2020 12:02:21  INFO basepage.py visiable_ele [ line:38 ] 元素('id', 'com.xxzb.fenwoo:id/et_phone')查找成功，耗时0.06980633735656738s
 Thu, 05 Mar 2020 12:02:21  INFO basepage.py get_ele [ line:56 ] 元素('id', 'com.xxzb.fenwoo:id/et_phone')查找成功！
 Thu, 05 Mar 2020 12:02:22  INFO basepage.py input_ele_text [ line:81 ] 元素('id', 'com.xxzb.fenwoo:id/et_phone')成功输入99999999999
 Thu, 05 Mar 2020 12:02:23  INFO basepage.py visiable_ele [ line:38 ] 元素('id', 'com.xxzb.fenwoo:id/btn_next_step')查找成功，耗时0.8081274032592773s
 Thu, 05 Mar 2020 12:02:23  INFO basepage.py get_ele [ line:56 ] 元素('id', 'com.xxzb.fenwoo:id/btn_next_step')查找成功！
 Thu, 05 Mar 2020 12:02:23  INFO basepage.py click_ele [ line:102 ] 元素('id', 'com.xxzb.fenwoo:id/btn_next_step')成功点击!
 Thu, 05 Mar 2020 12:02:25  INFO basepage.py visiable_ele [ line:38 ] 元素('id', 'com.xxzb.fenwoo:id/tv_message')查找成功，耗时0.043923139572143555s
 Thu, 05 Mar 2020 12:02:25  INFO basepage.py get_ele [ line:56 ] 元素('id', 'com.xxzb.fenwoo:id/tv_message')查找成功！
 Thu, 05 Mar 2020 12:02:25  INFO basepage.py get_ele_attr [ line:113 ] 获取元素('id', 'com.xxzb.fenwoo:id/tv_message')得text属性为无效的手机号
 Thu, 05 Mar 2020 12:02:25  INFO test_0_login.py test_login_tele_wrong [ line:37 ] Test Pass!
 Thu, 05 Mar 2020 12:02:25  INFO basepage.py visiable_ele [ line:38 ] 元素('id', 'com.xxzb.fenwoo:id/btn_confirm')查找成功，耗时0.05856060981750488s
 Thu, 05 Mar 2020 12:02:25  INFO basepage.py get_ele [ line:56 ] 元素('id', 'com.xxzb.fenwoo:id/btn_confirm')查找成功！
 Thu, 05 Mar 2020 12:02:25  INFO basepage.py click_ele [ line:102 ] 元素('id', 'com.xxzb.fenwoo:id/btn_confirm')成功点击!
 Thu, 05 Mar 2020 12:02:46  INFO test_0_login.py test_login_success [ line:48 ] 成功登录用例
 Thu, 05 Mar 2020 12:02:46  INFO bottom_widget.py switch_bottom_widget [ line:15 ] 切换到页面我
 Thu, 05 Mar 2020 12:02:47  INFO basepage.py visiable_ele [ line:38 ] 元素('-android uiautomator', 'new UiSelector().text("我")')查找成功，耗时1.7998619079589844s
 Thu, 05 Mar 2020 12:02:48  INFO basepage.py visiable_ele [ line:38 ] 元素('-android uiautomator', 'new UiSelector().text("我")')查找成功，耗时0.0829622745513916s
 Thu, 05 Mar 2020 12:02:48  INFO basepage.py get_ele [ line:56 ] 元素('-android uiautomator', 'new UiSelector().text("我")')查找成功！
 Thu, 05 Mar 2020 12:02:49  INFO basepage.py click_ele [ line:102 ] 元素('-android uiautomator', 'new UiSelector().text("我")')成功点击!
 Thu, 05 Mar 2020 12:02:49  INFO basepage.py visiable_ele [ line:38 ] 元素('id', 'com.xxzb.fenwoo:id/et_phone')查找成功，耗时0.5171504020690918s
 Thu, 05 Mar 2020 12:02:50  INFO basepage.py visiable_ele [ line:38 ] 元素('id', 'com.xxzb.fenwoo:id/et_phone')查找成功，耗时0.4239215850830078s
 Thu, 05 Mar 2020 12:02:50  INFO basepage.py get_ele [ line:56 ] 元素('id', 'com.xxzb.fenwoo:id/et_phone')查找成功！
 Thu, 05 Mar 2020 12:02:50  INFO basepage.py input_ele_text [ line:81 ] 元素('id', 'com.xxzb.fenwoo:id/et_phone')成功输入18684720553
 Thu, 05 Mar 2020 12:02:51  INFO basepage.py visiable_ele [ line:38 ] 元素('id', 'com.xxzb.fenwoo:id/btn_next_step')查找成功，耗时0.8256990909576416s
 Thu, 05 Mar 2020 12:02:51  INFO basepage.py get_ele [ line:56 ] 元素('id', 'com.xxzb.fenwoo:id/btn_next_step')查找成功！
 Thu, 05 Mar 2020 12:02:51  INFO basepage.py click_ele [ line:102 ] 元素('id', 'com.xxzb.fenwoo:id/btn_next_step')成功点击!
 Thu, 05 Mar 2020 12:02:53  INFO basepage.py visiable_ele [ line:38 ] 元素('id', 'com.xxzb.fenwoo:id/et_pwd')查找成功，耗时1.9444642066955566s
 Thu, 05 Mar 2020 12:02:53  INFO basepage.py visiable_ele [ line:38 ] 元素('id', 'com.xxzb.fenwoo:id/et_pwd')查找成功，耗时0.061487674713134766s
 Thu, 05 Mar 2020 12:02:53  INFO basepage.py get_ele [ line:56 ] 元素('id', 'com.xxzb.fenwoo:id/et_pwd')查找成功！
 Thu, 05 Mar 2020 12:02:56  INFO basepage.py input_ele_text [ line:81 ] 元素('id', 'com.xxzb.fenwoo:id/et_pwd')成功输入python
 Thu, 05 Mar 2020 12:02:57  INFO basepage.py visiable_ele [ line:38 ] 元素('id', 'com.xxzb.fenwoo:id/btn_next_step')查找成功，耗时1.339717149734497s
 Thu, 05 Mar 2020 12:02:58  INFO basepage.py get_ele [ line:56 ] 元素('id', 'com.xxzb.fenwoo:id/btn_next_step')查找成功！
 Thu, 05 Mar 2020 12:02:58  INFO basepage.py click_ele [ line:102 ] 元素('id', 'com.xxzb.fenwoo:id/btn_next_step')成功点击!
 Thu, 05 Mar 2020 12:03:06  INFO bottom_widget.py switch_bottom_widget [ line:15 ] 切换到页面首页
 Thu, 05 Mar 2020 12:03:06  ERROR basepage.py imgs_screenshot [ line:129 ] 截图失败!
Traceback (most recent call last):
  File "d:\python\mystudy\AppFramework\Common\basepage.py", line 36, in visiable_ele
    WebDriverWait(self.driver,timeout=20,poll_frequency=0.01).until(EC.visibility_of_element_located(loc))
  File "D:\Program Files (x86)\python\lib\site-packages\selenium\webdriver\support\wait.py", line 71, in until
    value = method(self._driver)
  File "D:\Program Files (x86)\python\lib\site-packages\selenium\webdriver\support\expected_conditions.py", line 128, in __call__
    return _element_if_visible(_find_element(driver, self.locator))
  File "D:\Program Files (x86)\python\lib\site-packages\selenium\webdriver\support\expected_conditions.py", line 415, in _find_element
    raise e
  File "D:\Program Files (x86)\python\lib\site-packages\selenium\webdriver\support\expected_conditions.py", line 411, in _find_element
    return driver.find_element(*by)
  File "D:\Program Files (x86)\python\lib\site-packages\appium\webdriver\webdriver.py", line 282, in find_element
    return self.execute(RemoteCommand.FIND_ELEMENT, {
  File "D:\Program Files (x86)\python\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "D:\Program Files (x86)\python\lib\site-packages\appium\webdriver\errorhandler.py", line 29, in check_response
    raise wde
  File "D:\Program Files (x86)\python\lib\site-packages\appium\webdriver\errorhandler.py", line 24, in check_response
    super(MobileErrorHandler, self).check_response(response)
  File "D:\Program Files (x86)\python\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.WebDriverException: Message: An unknown server-side error occurred while processing the command. Original error: Could not proxy command to remote server. Original error: Error: socket hang up


During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "d:\python\mystudy\AppFramework\Common\basepage.py", line 126, in imgs_screenshot
    self.driver.save_screenshot(pic_path)
  File "D:\Program Files (x86)\python\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 1055, in save_screenshot
    return self.get_screenshot_as_file(filename)
  File "D:\Program Files (x86)\python\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 1032, in get_screenshot_as_file
    png = self.get_screenshot_as_png()
  File "D:\Program Files (x86)\python\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 1064, in get_screenshot_as_png
    return base64.b64decode(self.get_screenshot_as_base64().encode('ascii'))
  File "D:\Program Files (x86)\python\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 1074, in get_screenshot_as_base64
    return self.execute(Command.SCREENSHOT)['value']
  File "D:\Program Files (x86)\python\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "D:\Program Files (x86)\python\lib\site-packages\appium\webdriver\errorhandler.py", line 29, in check_response
    raise wde
  File "D:\Program Files (x86)\python\lib\site-packages\appium\webdriver\errorhandler.py", line 24, in check_response
    super(MobileErrorHandler, self).check_response(response)
  File "D:\Program Files (x86)\python\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.WebDriverException: Message: An unknown server-side error occurred while processing the command. Original error: Could not proxy command to remote server. Original error: Error: socket hang up

 Thu, 05 Mar 2020 12:03:06  ERROR test_0_login.py test_login_success [ line:62 ] 登录失败!
Traceback (most recent call last):
  File "d:\python\mystudy\AppFramework\Common\basepage.py", line 36, in visiable_ele
    WebDriverWait(self.driver,timeout=20,poll_frequency=0.01).until(EC.visibility_of_element_located(loc))
  File "D:\Program Files (x86)\python\lib\site-packages\selenium\webdriver\support\wait.py", line 71, in until
    value = method(self._driver)
  File "D:\Program Files (x86)\python\lib\site-packages\selenium\webdriver\support\expected_conditions.py", line 128, in __call__
    return _element_if_visible(_find_element(driver, self.locator))
  File "D:\Program Files (x86)\python\lib\site-packages\selenium\webdriver\support\expected_conditions.py", line 415, in _find_element
    raise e
  File "D:\Program Files (x86)\python\lib\site-packages\selenium\webdriver\support\expected_conditions.py", line 411, in _find_element
    return driver.find_element(*by)
  File "D:\Program Files (x86)\python\lib\site-packages\appium\webdriver\webdriver.py", line 282, in find_element
    return self.execute(RemoteCommand.FIND_ELEMENT, {
  File "D:\Program Files (x86)\python\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "D:\Program Files (x86)\python\lib\site-packages\appium\webdriver\errorhandler.py", line 29, in check_response
    raise wde
  File "D:\Program Files (x86)\python\lib\site-packages\appium\webdriver\errorhandler.py", line 24, in check_response
    super(MobileErrorHandler, self).check_response(response)
  File "D:\Program Files (x86)\python\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.WebDriverException: Message: An unknown server-side error occurred while processing the command. Original error: Could not proxy command to remote server. Original error: Error: socket hang up


During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\python\mystudy\AppFramework\TestCases\test_0_login.py", line 59, in test_login_success
    assert not IndexPage(init_keep,init_session,name).visiable_login_btn()
  File "d:\python\mystudy\AppFramework\PageObjects\indexpage.py", line 21, in visiable_login_btn
    BottomWidget(self.driver,self.logging,self.model).switch_bottom_widget('首页')
  File "d:\python\mystudy\AppFramework\PageObjects\bottom_widget.py", line 17, in switch_bottom_widget
    self.visiable_ele(loc)
  File "d:\python\mystudy\AppFramework\Common\basepage.py", line 40, in visiable_ele
    self.imgs_screenshot()
  File "d:\python\mystudy\AppFramework\Common\basepage.py", line 130, in imgs_screenshot
    raise e
  File "d:\python\mystudy\AppFramework\Common\basepage.py", line 126, in imgs_screenshot
    self.driver.save_screenshot(pic_path)
  File "D:\Program Files (x86)\python\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 1055, in save_screenshot
    return self.get_screenshot_as_file(filename)
  File "D:\Program Files (x86)\python\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 1032, in get_screenshot_as_file
    png = self.get_screenshot_as_png()
  File "D:\Program Files (x86)\python\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 1064, in get_screenshot_as_png
    return base64.b64decode(self.get_screenshot_as_base64().encode('ascii'))
  File "D:\Program Files (x86)\python\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 1074, in get_screenshot_as_base64
    return self.execute(Command.SCREENSHOT)['value']
  File "D:\Program Files (x86)\python\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "D:\Program Files (x86)\python\lib\site-packages\appium\webdriver\errorhandler.py", line 29, in check_response
    raise wde
  File "D:\Program Files (x86)\python\lib\site-packages\appium\webdriver\errorhandler.py", line 24, in check_response
    super(MobileErrorHandler, self).check_response(response)
  File "D:\Program Files (x86)\python\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.WebDriverException: Message: An unknown server-side error occurred while processing the command. Original error: Could not proxy command to remote server. Original error: Error: socket hang up

 Thu, 05 Mar 2020 12:03:55  INFO test_0_login.py test_login_tele_wrong [ line:30 ] 手机号输入错误用例
 Thu, 05 Mar 2020 12:03:55  INFO bottom_widget.py switch_bottom_widget [ line:15 ] 切换到页面我
 Thu, 05 Mar 2020 12:03:56  INFO basepage.py visiable_ele [ line:38 ] 元素('-android uiautomator', 'new UiSelector().text("我")')查找成功，耗时1.296006679534912s
 Thu, 05 Mar 2020 12:03:56  INFO basepage.py visiable_ele [ line:38 ] 元素('-android uiautomator', 'new UiSelector().text("我")')查找成功，耗时0.11809754371643066s
 Thu, 05 Mar 2020 12:03:57  INFO basepage.py get_ele [ line:56 ] 元素('-android uiautomator', 'new UiSelector().text("我")')查找成功！
 Thu, 05 Mar 2020 12:03:57  INFO basepage.py click_ele [ line:102 ] 元素('-android uiautomator', 'new UiSelector().text("我")')成功点击!
 Thu, 05 Mar 2020 12:03:58  INFO basepage.py visiable_ele [ line:38 ] 元素('id', 'com.xxzb.fenwoo:id/et_phone')查找成功，耗时1.1010513305664062s
 Thu, 05 Mar 2020 12:03:58  INFO basepage.py visiable_ele [ line:38 ] 元素('id', 'com.xxzb.fenwoo:id/et_phone')查找成功，耗时0.061487674713134766s
 Thu, 05 Mar 2020 12:03:59  INFO basepage.py get_ele [ line:56 ] 元素('id', 'com.xxzb.fenwoo:id/et_phone')查找成功！
 Thu, 05 Mar 2020 12:04:00  INFO basepage.py input_ele_text [ line:81 ] 元素('id', 'com.xxzb.fenwoo:id/et_phone')成功输入 
 Thu, 05 Mar 2020 12:04:01  INFO basepage.py visiable_ele [ line:38 ] 元素('id', 'com.xxzb.fenwoo:id/btn_next_step')查找成功，耗时0.8353478908538818s
 Thu, 05 Mar 2020 12:04:01  INFO basepage.py get_ele [ line:56 ] 元素('id', 'com.xxzb.fenwoo:id/btn_next_step')查找成功！
 Thu, 05 Mar 2020 12:04:01  INFO basepage.py click_ele [ line:102 ] 元素('id', 'com.xxzb.fenwoo:id/btn_next_step')成功点击!
 Thu, 05 Mar 2020 12:04:04  INFO basepage.py visiable_ele [ line:38 ] 元素('id', 'com.xxzb.fenwoo:id/tv_message')查找成功，耗时1.0209293365478516s
 Thu, 05 Mar 2020 12:04:04  INFO basepage.py get_ele [ line:56 ] 元素('id', 'com.xxzb.fenwoo:id/tv_message')查找成功！
 Thu, 05 Mar 2020 12:04:04  INFO basepage.py get_ele_attr [ line:113 ] 获取元素('id', 'com.xxzb.fenwoo:id/tv_message')得text属性为无效的手机号
 Thu, 05 Mar 2020 12:04:04  INFO test_0_login.py test_login_tele_wrong [ line:37 ] Test Pass!
 Thu, 05 Mar 2020 12:04:04  INFO basepage.py visiable_ele [ line:38 ] 元素('id', 'com.xxzb.fenwoo:id/btn_confirm')查找成功，耗时0.05126833915710449s
 Thu, 05 Mar 2020 12:04:04  INFO basepage.py get_ele [ line:56 ] 元素('id', 'com.xxzb.fenwoo:id/btn_confirm')查找成功！
 Thu, 05 Mar 2020 12:04:04  INFO basepage.py click_ele [ line:102 ] 元素('id', 'com.xxzb.fenwoo:id/btn_confirm')成功点击!
 Thu, 05 Mar 2020 12:04:24  INFO test_0_login.py test_login_tele_wrong [ line:30 ] 手机号输入错误用例
 Thu, 05 Mar 2020 12:04:24  INFO bottom_widget.py switch_bottom_widget [ line:15 ] 切换到页面我
 Thu, 05 Mar 2020 12:04:26  INFO basepage.py visiable_ele [ line:38 ] 元素('-android uiautomator', 'new UiSelector().text("我")')查找成功，耗时2.084517002105713s
 Thu, 05 Mar 2020 12:04:26  INFO basepage.py visiable_ele [ line:38 ] 元素('-android uiautomator', 'new UiSelector().text("我")')查找成功，耗时0.11321568489074707s
 Thu, 05 Mar 2020 12:04:26  INFO basepage.py get_ele [ line:56 ] 元素('-android uiautomator', 'new UiSelector().text("我")')查找成功！
 Thu, 05 Mar 2020 12:04:27  INFO basepage.py click_ele [ line:102 ] 元素('-android uiautomator', 'new UiSelector().text("我")')成功点击!
 Thu, 05 Mar 2020 12:04:29  INFO basepage.py visiable_ele [ line:38 ] 元素('id', 'com.xxzb.fenwoo:id/et_phone')查找成功，耗时1.2922825813293457s
 Thu, 05 Mar 2020 12:04:29  INFO basepage.py visiable_ele [ line:38 ] 元素('id', 'com.xxzb.fenwoo:id/et_phone')查找成功，耗时0.0790567398071289s
 Thu, 05 Mar 2020 12:04:29  INFO basepage.py get_ele [ line:56 ] 元素('id', 'com.xxzb.fenwoo:id/et_phone')查找成功！
 Thu, 05 Mar 2020 12:04:29  INFO basepage.py input_ele_text [ line:81 ] 元素('id', 'com.xxzb.fenwoo:id/et_phone')成功输入15264
 Thu, 05 Mar 2020 12:04:30  INFO basepage.py visiable_ele [ line:38 ] 元素('id', 'com.xxzb.fenwoo:id/btn_next_step')查找成功，耗时0.8628714084625244s
 Thu, 05 Mar 2020 12:04:30  INFO basepage.py get_ele [ line:56 ] 元素('id', 'com.xxzb.fenwoo:id/btn_next_step')查找成功！
 Thu, 05 Mar 2020 12:04:30  INFO basepage.py click_ele [ line:102 ] 元素('id', 'com.xxzb.fenwoo:id/btn_next_step')成功点击!
 Thu, 05 Mar 2020 12:04:32  INFO basepage.py visiable_ele [ line:38 ] 元素('id', 'com.xxzb.fenwoo:id/tv_message')查找成功，耗时0.046851158142089844s
 Thu, 05 Mar 2020 12:04:32  INFO basepage.py get_ele [ line:56 ] 元素('id', 'com.xxzb.fenwoo:id/tv_message')查找成功！
 Thu, 05 Mar 2020 12:04:32  INFO basepage.py get_ele_attr [ line:113 ] 获取元素('id', 'com.xxzb.fenwoo:id/tv_message')得text属性为无效的手机号
 Thu, 05 Mar 2020 12:04:32  INFO test_0_login.py test_login_tele_wrong [ line:37 ] Test Pass!
 Thu, 05 Mar 2020 12:04:33  INFO basepage.py visiable_ele [ line:38 ] 元素('id', 'com.xxzb.fenwoo:id/btn_confirm')查找成功，耗时0.06343889236450195s
 Thu, 05 Mar 2020 12:04:33  INFO basepage.py get_ele [ line:56 ] 元素('id', 'com.xxzb.fenwoo:id/btn_confirm')查找成功！
 Thu, 05 Mar 2020 12:04:33  INFO basepage.py click_ele [ line:102 ] 元素('id', 'com.xxzb.fenwoo:id/btn_confirm')成功点击!
 Thu, 05 Mar 2020 12:04:58  INFO test_0_login.py test_login_tele_wrong [ line:30 ] 手机号输入错误用例
 Thu, 05 Mar 2020 12:04:58  INFO bottom_widget.py switch_bottom_widget [ line:15 ] 切换到页面我
 Thu, 05 Mar 2020 12:04:59  INFO basepage.py visiable_ele [ line:38 ] 元素('-android uiautomator', 'new UiSelector().text("我")')查找成功，耗时1.0218696594238281s
 Thu, 05 Mar 2020 12:04:59  INFO basepage.py visiable_ele [ line:38 ] 元素('-android uiautomator', 'new UiSelector().text("我")')查找成功，耗时0.11516857147216797s
 Thu, 05 Mar 2020 12:04:59  INFO basepage.py get_ele [ line:56 ] 元素('-android uiautomator', 'new UiSelector().text("我")')查找成功！
 Thu, 05 Mar 2020 12:05:00  INFO basepage.py click_ele [ line:102 ] 元素('-android uiautomator', 'new UiSelector().text("我")')成功点击!
 Thu, 05 Mar 2020 12:05:01  INFO basepage.py visiable_ele [ line:38 ] 元素('id', 'com.xxzb.fenwoo:id/et_phone')查找成功，耗时0.5085339546203613s
 Thu, 05 Mar 2020 12:05:01  INFO basepage.py visiable_ele [ line:38 ] 元素('id', 'com.xxzb.fenwoo:id/et_phone')查找成功，耗时0.07222390174865723s
 Thu, 05 Mar 2020 12:05:01  INFO basepage.py get_ele [ line:56 ] 元素('id', 'com.xxzb.fenwoo:id/et_phone')查找成功！
 Thu, 05 Mar 2020 12:05:01  INFO basepage.py input_ele_text [ line:81 ] 元素('id', 'com.xxzb.fenwoo:id/et_phone')成功输入99999999999
 Thu, 05 Mar 2020 12:05:03  INFO basepage.py visiable_ele [ line:38 ] 元素('id', 'com.xxzb.fenwoo:id/btn_next_step')查找成功，耗时1.3615503311157227s
 Thu, 05 Mar 2020 12:05:03  INFO basepage.py get_ele [ line:56 ] 元素('id', 'com.xxzb.fenwoo:id/btn_next_step')查找成功！
 Thu, 05 Mar 2020 12:05:03  INFO basepage.py click_ele [ line:102 ] 元素('id', 'com.xxzb.fenwoo:id/btn_next_step')成功点击!
